django/thunderstore/community/views/community.py

```diff
@@ -0,0 +1,85 @@
+from django.conf import settings
+from django.db.models import BigIntegerField, Count, Sum
+from django.shortcuts import redirect
+from django.views import View
+from django.views.generic.list import ListView
+
+from thunderstore.cache.cache import CacheBustCondition
+from thunderstore.cache.pagination import CachedPaginator
+from thunderstore.community.models.community_site import CommunitySite
+from thunderstore.repository.models.package import Package
+
+# Should be divisible by 4 and 3
+MODS_PER_PAGE = 24
+
+
+class CommunitiesView(ListView):
+    model = CommunitySite
+    paginate_by = MODS_PER_PAGE
+    paginator_class = CachedPaginator
+    template_name = "community/communities_list.html"
+
+    def get_base_queryset(self):
+        return self.model.objects.exclude(is_listed=False)
+
+    def get_page_title(self):
+        return "Communities"
+
+    def get_cache_vary(self):
+        return "communities"
+
+    def get_full_cache_vary(self):
```

Comment:
1. This could be just `return self.get_cache_vary()`
2. I'm not entirely sure if these methods are used in some magic, but since it seems they're only used to generate cache key for the paginator, I don't quite see the point of having two separate methods. I **assume** the existing package list views have split these into two methods so inherited views can implement just one of the methods
3. If the above is correct, there's not much point in having a method that returns a static string, it could just be hardcoded to the paginator function call.

Reply:
Yep you are correct. Separate functions right here are redundant. I'll fix